fixtures/most_popular-options.proto

```diff
@@ -6,38 +6,38 @@ import "google/api/annotations.proto";
 
 package themostpopularapi;
 
-message GetMostemailedSectionTimePeriodRequest {
+message GETMostemailedSectionTimePeriodJsonRequest {
     string Accept = 1;
     string api_key = 2;
     string section = 3;
     string time_period = 4;
 }
 
-message GetMostemailedSectionTimePeriodResponse {
+message GETMostemailedSectionTimePeriodJsonResponse {
     string copyright = 1;
     int32 num_results = 2;
     repeated ArticleWithCountType results = 3;
     string status = 4;
 }
 
-message GetMostsharedSectionTimePeriodRequest {
+message GETMostsharedSectionTimePeriodJsonRequest {
```

Comment:
Hrm. I just noticed the change from `Get` to `GET` for these struct names.

Any chance we can leave it as the title case?

Also noticed the `Json` suffixes are sneaking in again. Can we keep those trimmed?

Reply:
This message name was generated from operation-id: GET_mostshared-section-time-period-json, that is why there is a GET and Json suffix.

I can make all words titled and remove any Json suffixes. Is that ok ?

Reply:
That's be great. The original implementation also title cases all words it sees: https://github.com/NYTimes/openapi2proto/blob/master/openapi.go#L445-L447

Reply:
Done.